pkgs/applications/misc/opencpn/default.nix

```diff
@@ -48,9 +51,32 @@ stdenv.mkDerivation rec {
     hash = "sha256-sNZYf/2gtjRrrGPuazVnKTgcuIQpKPazhexqlK21T4g=";
   };
 
-  nativeBuildInputs = [ cmake lsb-release pkg-config ];
+  patches = [
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/OpenCPN/OpenCPN/pull/2473.patch";
```

Comment:
Since the PR was merged, please use the URL from the commit to master:

https://github.com/OpenCPN/OpenCPN/commit/30fa16850ba97d3df0622273947e3e3975b8e6c0.patch